src/main/java/uk/gov/justice/hmpps/prison/service/BookingService.java

```diff
@@ -342,16 +326,32 @@ public Map<Long, PrivilegeSummary> getBookingIEPSummary(final List<Long> booking
         // If no IEP details exist for offender, cannot derive an IEP summary.
         bookingIds.stream()
                 .filter(bookingId -> !mapOfEip.containsKey(bookingId))
-                .collect(toList())
-                .forEach(bookingId -> mapOfEip.put(bookingId, PrivilegeSummary.builder()
-                        .bookingId(bookingId)
-                        .iepLevel(defaultIepLevel)
-                        .iepDetails(Collections.emptyList())
-                        .build()));
+                .toList()
+                .forEach(bookingId -> {
+                    final var offenderBooking = offenderBookingRepository.findById(bookingId).orElseThrow(EntityNotFoundException.withId(bookingId));
```

Comment:
Not ideal doing another lookup but it's needed so we can get a prison's default IEP. This is an edge case so not expecting it to called frequently